app/views/shared/_navbar.html.erb

```diff
@@ -5,7 +5,7 @@
   <div>
     <ul class="flex space-x-14 text-base text-black">
       <li><a href="/" class="hover:text-gray-600">Home</a></li>
-      <li><a href="#" class="hover:text-gray-600">Mentor</a></li>
+      <li><a href="/dashboard/mentee" class="hover:text-gray-600">Mentor</a></li>

```

Comment:
I think the name "Mentor" in this situation should only be shown when the current user (mentee) is matched.